src/scripts/admin/admin.users.jsx

```diff
@@ -104,8 +133,18 @@ let users = React.createClass({
 
     _inputChange(e) {
         actions.inputChange('newUserForm', e.target.name, e.target.value);
-    }
+    },
 
+    _searchUser(e) {
+        this.setState({filterAdminActive: false, inputValue: e.target.value});
+        actions.searchUser(e.target.value);
+    },
+
+    _filterAdmin(){
+        let filterAdminActive = !this.state.filterAdminActive;
+        actions.filterAdmin();
+        this.setState({filterAdminActive, inputValue: ''});
+    }
```

Comment:
In both _searchUser and _filterAdmin you shouldn't need to set the state here. When a component is listening to a store you should do all state management in the store itself. Otherwise the component can end up with state the store isn't aware of and in these cases it looks like the store should already have the state data you're updating here.